README.md

```diff
@@ -9,7 +9,7 @@
 This guide will cover the fundamentals of building your first Extension. It will allow Twitch viewers to cycle through colors via a visual element added on top of the video player (we call this a component overlay). On the frontend, a user clicks a button in this component that changes the color of a circle. Instead of changing the CSS locally, it makes a call to a backend server for a new hex value (we call this an Extension Backend Service or EBS for short).
 
 ## What's the Twitch Developer Rig?
-<img src="https://github.com/twitchdev/extension-getting-started/blob/master/Extension-Rig-Banner.png" width="45%">
+<img src="https://github.com/twitchdev/extension-getting-started/raw/main/Extension-Rig-Banner.png" width="45%">
```

Comment:
This change is an improvement: it works in the PR view where the existing one doesn't. :+1: 

But I think we can simplify it further. Since the image is stored in the same location as this README, we should be able to use a relative link.
```suggestion
<img src="Extension-Rig-Banner.png" width="45%">
```

This frees us from worrying about which host github is using for images or what the branch is named.